docs/tools/nft-catalog/index.mdx

```diff
@@ -4,11 +4,28 @@ sidebar_label: NFT Catalog
 description: The NFT Catalog is an on chain registry listing NFT collections that exists on Flow which adhere to the NFT metadata standard.
 ---
 
-The NFT Catalog is an on chain registry listing NFT collections that exists on Flow which adhere to the NFT metadata standard. This empowers dApp developers to easily build on top of and discover interoperable NFT collections on Flow.
+<Callout type="info">
+
+The NFT Catalog is in the process of being deprecated.
+You can still use it for existing collections in the Catalog,
+but no new collections will be added from now on.
+Most of the functionality that the Catalog provided is now required
+and handled by default by the NonFungibleToken standard with Metadata Views.
+
+Additionally, [TokenList](https://token-list.fixes.world) is a new permissionless tool that
```

Comment:
Is there existing documentation or guides we can direct readers to as well?

Reply:
@btspoony Do we have documentation for TokenList we can point people to?

Reply:
It is very simple, basically all the information is included on the Landing Page.

Reply:
The only doc is https://docs.fixes.world/concepts/token-list